percona/controller/pgbackup/controller.go

```diff
@@ -75,11 +77,19 @@ func (r *PGBackupReconciler) Reconcile(ctx context.Context, request reconcile.Re
 
 	pgBackup.Default()
 
+	if pgBackup.DeletionTimestamp != nil {
```

Comment:
Here's maybe it is preferable to is `..isZero()` instead of nil checking to catch also the case that for some reason the deletion timestamp wasn't initialized to the zero time instant